scripts/pre-publish.js

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Silence Laboratories Pte. Ltd.
+// This software is licensed under the Silence Laboratories License Agreement.
+
+const chalk = require('chalk');
+const fs = require('fs');
+
+// Read the package.json file
+const packageJson = JSON.parse(fs.readFileSync('package.json', 'utf8'));
+const snapManifestJson = JSON.parse(fs.readFileSync('snap.manifest.json', 'utf8'));
+
+const execEnv = process.env.NODE_ENV;
+const currentVersion = packageJson.version;
+let baseRepoUrl = '';
+let name = '';
+let rpcOrigins = [];
+let keyringOrigins = [];
+
+// Check the environment
+if (execEnv === 'staging') {
+	console.log(`${chalk.green.bold('PREPUBLISH STAGING ANNOUNCEMENT')}\u{1F447}\u{1F447}\u{1F447}`);
```

Comment:
what is that hardcoded byted?\u{1F447}\u{1F447}\u{1F447}

Reply:
https://www.compart.com/en/unicode/U+1F447 That's the icon to make the stdout looks nice